internal/pkg/pkg.go

```diff
@@ -459,7 +445,7 @@ func IsPackageDir(path string) (bool, error) {
 // information, it will always return false.
 // If a Kptfile is not found on the provided path, an error will be returned.
 func IsPackageUnfetched(path string) (bool, error) {
-	kf, err := ReadKptfile(path)
+	kf, err := ReadKptfile(filesys.FileSystemOrOnDisk{}, path)
```

Comment:
This method should be provided with `fsys`.

Reply:
We can do these in next iterations when we are dealing with pkg workflows.